wtforms/fields/core.py

```diff
@@ -154,6 +157,14 @@ def __call__(self, **kwargs):
         """
         return self.meta.render_field(self, kwargs)
 
+    @classmethod
+    def checkValidators(cls, validators):
+        if validators is not None:
+            for validator in list(validators):
+                if inspect.isclass(validator) or not hasattr(validator, '__call__'):
```

Comment:
`if callable(validator)` and not inspect.isclass(validator):`

Reply:
Perhaps also split these checks and have different messages: "{} is not a valid validator because it is not callable." and "{} is not a valid validator because it is a class, it should be an instance."